src/main.c

```diff
@@ -31,6 +47,17 @@ usage(void)
 	exit(0);
 }
 
+static void
+send_signal_to_labwc_pid(int signal)
+{
+	char *pid = getenv("LABWC_PID");
+	if (!pid) {
+		wlr_log(WLR_ERROR, "LABWC_PID not set");
+		exit(1);
+	}
+	kill(atoi(pid), signal);
```

Comment:
This might call `kill(0, signal)` when `atoi()` fails to parse the content which is a special case:

> If pid equals 0, then sig is sent to every process in the process group of the calling process.